src/Modules/MotdModule/Controllers/MotdCommandController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using EvoSC.Commands.Attributes;
+using EvoSC.Commands.Interfaces;
+using EvoSC.Common.Controllers;
+using EvoSC.Common.Controllers.Attributes;
+using EvoSC.Modules.Official.MotdModule.Interfaces;
+
+namespace EvoSC.Modules.Official.MotdModule.Controllers;
+
+[Controller]
+public class MotdCommandController : EvoScController<ICommandInteractionContext>
+{
+    private readonly IMotdService _motdService;
+
+    public MotdCommandController(IMotdService motdService)
+    {
+        _motdService = motdService;
+    }
+
+    [ChatCommand("motd", "[Command.OpenMotd]")]
+    public async Task OpenMotdAsync()
+        => await _motdService.ShowAsync(Context.Player);
+    
+    [ChatCommand("motdsetinterval", "[Command.MotdSetFetchInterval]", "MotdPermissions.SetFetchInterval")]
```

Comment:
```suggestion
    [ChatCommand("motdsetinterval", "[Command.MotdSetFetchInterval]", MotdPermissions.SetFetchInterval)]
```

You can use the enum directly here.